guides/common/attributes-base.adoc

```diff
@@ -7,7 +7,7 @@
 :InstallingProjectDocURL: {BaseURL}Installing_Server/{BaseFilenameURL}#
 :InstallingSmartProxyDocURL: {BaseURL}Installing_Proxy/{BaseFilenameURL}#
 :ManagingHostsDocURL: {BaseURL}Managing_Hosts/{BaseFilenameURL}#
-:ManagingConfigurationsPuppetDocURL: {BaseURL}Managing_Configurations_Using_Puppet/{BaseFilenameURL}#
+:ManagingConfigurationsPuppetDocURL: {BaseURL}Managing_Configurations_Puppet/{BaseFilenameURL}#
```

Comment:
Shouldn't it be different only in the Satellite attributes document?

Reply:
https://docs.theforeman.org/nightly/Managing_Configurations_Puppet/index-foreman-el.html - this is what is live on our site.

Reply:
Ah ok, that makes sense.